sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -691,11 +696,6 @@ public WritableColumnVector arrayData() {
    */
   protected abstract WritableColumnVector reserveNewColumn(int capacity, DataType type);
 
-  protected boolean isArray() {
-    return type instanceof ArrayType || type instanceof BinaryType || type instanceof StringType ||
```

Comment:
Would it be better to minimize this change? i.e. just `protected` -> `public` without changing the place.

Reply:
happy to keep the previous place but I noticed that all the `public` methods in this class are grouped together so I thought that I should move this to keep that consistent

Reply:
`WritableColumnVector` class will be public API in Spark 2.3. It seem to be necessary to discuss with @cloud-fan for changing the visibility of a method or field.